fat-jar/fat-jar-pom.xml.in

```diff
@@ -0,0 +1,677 @@
+<!-- Generated by Makefile on Tue Mar 3 17:23:42 EST 2015 -->
```

Comment:
Can we have just the one POM file with a profile for the FatJar? Maintenance would be much easier.